src/audio.c

```diff
@@ -163,7 +163,7 @@ void initAudio(void)
     (gDevice->want).callback = audioCallback;
     (gDevice->want).userdata = calloc(1, sizeof(Audio));
 
-    global = (gDevice->want).userdata;
+    global = (Audio *)(gDevice->want).userdata;
```

Comment:
`(Audio *) (gDevice->want).userdata;`

spacing

Reply:
Also on line 177, change comment to "newAudio" instead of new!

Reply:
Done and done.

Reply:
I don't think line 177 changes went through

Reply:
You are correct, I fixed it.

Reply:
I apologize for force pushing, I'm trying to keep the commit history somewhat clean.

Reply:
totally fine with me